clients/runtime/src/rpc.rs

```diff
@@ -313,13 +310,19 @@ impl SpacewalkParachain {
 	where
 		Address: StorageAddress<IsFetchable = Yes, IsDefaultable = Yes>,
 	{
-		Ok(self.api.storage().at_latest().await.unwrap().fetch_or_default(&address).await?)
+		Ok(self.get_latest_storage().await.fetch_or_default(&address).await?)
 	}
 
 	pub async fn get_finalized_block_hash(&self) -> Result<Option<H256>, Error> {
 		Ok(Some(self.api.backend().latest_finalized_block_ref().await?.hash()))
 	}
 
+	async fn get_latest_storage(
+		&self,
+	) -> Storage<SpacewalkRuntime, OnlineClient<SpacewalkRuntime>> {
+		self.api.storage().at_latest().await.expect("failed to get latest storage")
```

Comment:
Can we map this to an error instead of `expect`ing or would that require a lot of refactoring?

Reply:
I think we unfortunately cannot expect this to never fail, so it would be very good if we can handle this error gracefully.

Reply:
wrapped it inside `Error::SubxtRuntimeError()`